test/options.test.js

```diff
@@ -67,6 +67,33 @@ exports.mappingOverSomeAppliesFunctionToValue = function(test) {
     test.done();
 };
 
+
+exports.flatMapOnNoneToNone = function(test) {
```

Comment:
I'm suggesting that the existing test names aren't especially clear, and that writing in the style of the example I wrote above might be clearer.